src/OpenTK.Windowing.GraphicsLibraryFramework/Input/KeyboardState.cs

```diff
@@ -8,58 +8,54 @@
 //
 
 using System;
+using System.Collections;
 using System.Diagnostics;
 using System.Text;
+using OpenTK.Windowing.Common;
 
-namespace OpenTK.Windowing.Common.Input
+namespace OpenTK.Windowing.GraphicsLibraryFramework
 {
     /// <summary>
     /// Encapsulates the state of a Keyboard device.
     /// </summary>
-    public struct KeyboardState : IEquatable<KeyboardState>
+    public sealed class KeyboardState
```

Comment:
Once again, not a fan of sealed. 
Let users do what they want even it probably won't work, there might be one user who needs to extend this.

Reply:
We've already made JoystickState sealed, this is bringing the other input types in line with that.

Reply:
Ok I will propose removing those in my follow up pr :)